### ISSUE TITLE
Variant Descriptions synced to DEEP # instead of Greg or STC #s

### ISSUE THREAD

**Issue by ZacharyLesser** [upenndigitalscholarship/deep#92]
Right now the way Variant links work is that at the end of the Variant Description, the site automatically lists the variant Greg #s as links to the other record. For Collections, this is currently not working (outputting a bunch of "n/a"s), because Collections do not have Greg #s at all. On the old site, they were linked by STC/Wing # instead, but this has not been implemented on the new site.  

BUT in any case: one of the upgrades we want to make is to make this functionality uniform across all records by having all variants linked by DEEP # instead. (This will entail rewriting some of the static text of the Variant Descriptions but that is something we can do ourselves.)

So in other words, in the attached example from Hamlet, instead of saying "197b†", it would say "349" (the DEEP # of the *linked* record -- i.e., the "other" record).

[Hamlet variant desc.pdf](https://github.com/upenndigitalscholarship/deep/files/10461386/Hamlet.variant.desc.pdf)

And in the other example, from "Comedies, Histories, and Tragedies (1632)", instead of "n/a n/a. ..." it would say: "5083; 5084; 5085; 5086; 5087; 5088; 5089" 

In each case the DEEP # is a link to the variant record; multiple DEEP #s are separated by a semicolon and space.

[F2 variant desc.pdf](https://github.com/upenndigitalscholarship/deep/files/10461413/F2.variant.desc.pdf)
